lib/commands/network.js

```diff
@@ -42,6 +45,116 @@ commands.setNetworkConnection = async function (type) {
   return await this.getNetworkConnection();
 };
 
+commands.toggleData = async function () {
+  log.info('Toggling data');
```

Comment:
All of these "toggle" methods get their current state. It might be nice to log what that is, and what it is being toggled to.